src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java

```diff
@@ -1035,6 +1028,26 @@ public ActionForward refresh(ActionMapping mapping, ActionForm form, HttpServlet
         return actionForward;
     }
 
+    private String obtainDeliverToAddressForRefresh(IWantDocument iWantDocument) {
```

Comment:
Two thoughts on this:
1) These functions could be combined with something like findCampusAddressFromNetId(String netId)

2) It looks like this approach will over write a manually edited address with the campus address of the person.  Is that the desired outcome?

Reply:
Regarding your two points:

1. I have combined the two functions into one parameterized function as requested.
 
2. Per today's standup discussion with the functional user, the behavior of overwriting any manually entered address is the desired outcome.